hibernate-core/src/main/java/org/hibernate/type/ClobType.java

```diff
@@ -36,7 +39,21 @@ protected boolean registerUnderJavaType() {
 
 	@Override
 	protected Clob getReplacement(Clob original, Clob target, SharedSessionContractImplementor session) {
-		return session.getJdbcServices().getJdbcEnvironment().getDialect().getLobMergeStrategy().mergeClob( original, target, session );
+		if ( target == null ) {
+			return copyOriginalClob( (Clob) original, session );
```

Comment:
Same [comment](https://github.com/hibernate/hibernate-orm/pull/2390/files#r241243365) here.

Reply:
@gsmet, why not just delegate to the `LobMergeStrategy`?